index.ts

```diff
@@ -0,0 +1,60 @@
+const products = require("./data/products.json");
+
+interface Product {
+  name: string;
+  description: string;
+  price: number;
+  image: URL;
+  limit: number;
+  amount: number;
+}
+
+interface Cart {
+  products: Product[];
+}
+
+const addProduct = (cart: Cart, product: Product) => {
+  cart.products.push(product);
+};
+
+const removeProduct = (cart: Cart, productToRemove: Product) => {
+  cart.products.forEach((product, i) => {
```

Comment:
read about array functions.  use relevant function

Reply:
Changed to filter function